docs/source/contributors-guide/index.rst

```diff
@@ -0,0 +1,14 @@
+===================
+Contributor's Guide
+===================
+
+Do you want to help make ViRelAy even better? There many ways in which you can contribute to the project. This guide will explain everything you need to know to improve ViRelAy's code base and how to submit changes to us. Also, this guide will go into the details of ViRelAy's architecture and the file formats used to describe ViRelAy projects.
```

Comment:
We talked about this, but I am really not a fan of these super long lines. I would suggest that you break at least on sentence base, since full paragraph-lines are also not trivial to address in reviews.

Reply:
I am sorry, I am still not convinced. As I said, it is much worse to constantly re-break lines, which also produces horrible diffs.

Reply:
But that would not be the case for sentence-based line breaks?